pkgs/by-name/vm/vmctl/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  bash,
+  shellcheck,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "vmctl";
+  version = "1";
+
+  src = fetchFromGitHub {
+    owner = "SamsungDS";
+    repo = "vmctl";
+    rev = "5b6b7084b8cba06b474c0e020df965237f2c832c";
+    sha256 = "sha256-yDgaY2RJXBjWkMSQb4JaJzMGLFzowfOGixSRzzv2iIk=";
+  };
+
+  nativeBuildInputs = [
+    bash
+    shellcheck
```

Comment:
Looking at the upstream `Makefile`, it seems like this is used in the `check` target, which I think is built during our `checkPhase`. Maybe it would fit better in `nativeCheckInputs`?

Reply:
This is a good point. But the problem is, the check target is triggered even if I do just a `make`. I think I should patch the Makefile to not make it the default target somehow. I need check the Makefile syntax how to do it.

Reply:
Something like this:
```
diff --git a/Makefile b/Makefile
index b06e6f4..184bd20 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,11 @@
 SHELLCHECK := $(shell command -v shellcheck 2>/dev/null)
 
+default: all
+
+all:
+
+.PHONY: all check
+
 check:
 ifndef SHELLCHECK
        $(error "cannot find shellcheck; install to run check")
```

Reply:
Maybe the `buildPhase` isn't even needed? It's not like the makefile is doing anything else than running shellcheck. Maybe you could specify these?

```nix
dontBuild = true;
doCheck = true;
```

Reply:
Much better idea :D